CodeGeneration/Sources/SyntaxSupport/gyb_helpers/utils.py

```diff
@@ -98,3 +98,28 @@ def flat_documentation(indented_documentation):
   Creates a single-line documentation string from indented documentation as written in 'gyb_syntax_support'
   """
   return "".join(dedented_lines(indented_documentation)).replace("\n", "").replace('"','\\"').strip()
+
+def make_swift_attribute(attr):
+  """
+  
+  """
+  
+  class_name = type(attr).__name__
+  spaces = 10
+  parameters = ['name: "%s"' % attr.name]
+  
+  if hasattr(attr, 'class_name'):
+    parameters += ['className: "%s"' % attr.class_name]
+  
+  if hasattr(attr, 'options'):
+    mapped_options = map(lambda x: (' ' * (spaces + 8)) + '"%s"' % x, attr.options)
+    parameters += ['options: \n%s' % (',\n'.join(mapped_options))]
+  
+  if hasattr(attr, 'code'):
+    parameters += ['code: %s' % attr.code]
+  
+  if class_name not in ['ContextualSimpleDeclAttribute', 'ContextualDeclAttribute'] and hasattr(attr, 'swift_name'):
```

Comment:
Why do we need to explicitly exclude `ContextualSimpleDeclAttribute` and `ContextualDeclAttribute` here?

Reply:
Because those two don't use the `swift_name` in the parameters

https://github.com/apple/swift-syntax/blob/76d01195182593ff34f5ada1ab0910fae190fc9c/gyb_syntax_support/AttributeKinds.py#L29-L31

Reply:
Oh, mind adding that as a short explanation comment for the next one who stumbles over it?

Reply:
Added 💪